apps/webhook/README.md

```
@@ -184,35 +184,95 @@ Tests marked with `@pytest.mark.external` will be skipped automatically unless t
+
+# Utils
+from utils.url import normalize_url
+from utils.timing import timing_context
+from utils.text_processing import chunk_text
+
+# Config
+from config import Settings
+```
+
+**Key principle:** All imports are relative to `/apps/webhook/` (the root of this service). No `PYTHONPATH` manipulation required.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)